libs/portals/admin/ids-admin/src/components/Client/BasicInfo.tsx

```diff
@@ -1,48 +1,53 @@
-import { AccordionCard, Input, Stack } from '@island.is/island-ui/core'
+import React, { RefObject, useReducer, useRef } from 'react'
+
+import {
+  AccordionCard,
+  Input,
+  Stack,
+  toast,
+  Text,
+} from '@island.is/island-ui/core'
 import { useLocale } from '@island.is/localization'
-import React, { RefObject } from 'react'
+
 import { m } from '../../lib/messages'
 import ContentCard from '../../shared/components/ContentCard'
-import { toast } from '@island.is/island-ui/core'
+import { AuthAdminClientSecret } from './Client.loader'
 
 interface BasicInfoProps {
   clientId: string
-  clientSecret?: string
+  clientSecrets?: AuthAdminClientSecret | null
```

Comment:
You need to update the type here from the generated code to the following type
`GetClientQuery['authAdminClient']['environments'][0]['secrets']`

Reply:
Already was 😉